package.json

```diff
@@ -71,6 +71,7 @@
     "@npmcli/arborist": "7.5.2",
     "ajv": "^8.14.0",
     "ajv-formats": "^3.0.1",
+    "ajv-formats-draft2019": "^1.6.1",
```

Comment:
I am having better success with [validate-iri](https://www.npmjs.com/package/validate-iri) package.

```
import { validateIri, IriValidationStrategy } from 'validate-iri'
let yourIri = 'git@gitlab.com:behat-chrome/chrome-mink-driver.git'
console.log(validateIri(yourIri, IriValidationStrategy.Strict));

yourIri = '${repository.url}';
console.log(validateIri(yourIri, IriValidationStrategy.Strict));

yourIri = 'git+https://github.com/Alex-D/check-disk-space.git';
console.log(validateIri(yourIri, IriValidationStrategy.Strict));
```

Reply:
I was looking for some validation library but didn't find this one :+1: